charts/hono/templates/_helpers.tpl

```diff
@@ -479,18 +479,30 @@ The scope passed in is expected to be a dict with keys
 - (mandatory) "componentConfig": the component's configuration properties from the values.yaml file
 */}}
 {{- define "hono.component.healthChecks" }}
+{{- $global := .dot.Values -}}
+{{- $component := .componentConfig -}}
+{{- $probes := mergeOverwrite ( $global.probes | deepCopy ) ( $component.probes | default dict | deepCopy ) -}}
+
 livenessProbe:
   httpGet:
-    path: "/liveness"
-    port: "health"
-    scheme: "HTTP"
-  initialDelaySeconds: {{ default .dot.Values.livenessProbeInitialDelaySeconds .componentConfig.livenessProbeInitialDelaySeconds }}
+    path: {{ $probes.livenessProbe.httpGet.path | default "/liveness" }}
+    port: {{ $probes.livenessProbe.httpGet.port | default "health" }}
+    scheme: {{ $probes.livenessProbe.httpGet.scheme | default "HTTP" }}
+  periodSeconds: {{ $probes.livenessProbe.periodSeconds | default 10 }}
+  failureThreshold: {{ $probes.livenessProbe.failureThreshold | default 3 }}
+  initialDelaySeconds: {{ $probes.livenessProbe.initialDelaySeconds | default 300 }}
+  successThreshold: {{ $probes.livenessProbe.successThreshold | default 1 }}
+  timeoutSeconds: {{ $probes.livenessProbe.timeoutSeconds | default 1 }}
```

Comment:
IMHO we should increase the timeout here to e.g. 3 seconds to prevent unnecessary restarts of components, as 1s has proven to be hit quite often with Hono 2.0.0 deployments